gcloud/resource_manager/project.py

```diff
@@ -0,0 +1,323 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Define Projects."""
+
+from gcloud.exceptions import MissingClientError
+from gcloud.exceptions import NotFound
+
+
+class Project(object):
+    """Projects are containers for your work on Google Cloud Platform.
+
+    .. note::
+
+        It's unlikely that you'd need to instantiate this outside the context
+        of a :class:`.client.Client`, so in general, it's best to get a Project
+        from a Resource Manager client.
```

Comment:
"It's unlikely" and "it's best to get" are more than just informational. That's my point.

There is no best, it's up to the user. This is why I suggested

```
You can construct a :class:`Project` directly via the :method:`.client.Client.project` factory.
```